### ISSUE TITLE
Cannot read property 'x' of null

### ISSUE THREAD

**Issue by Chrisriis** [avocode/avocode#451]
IMPORTANT: Please try to describe how did it happen.

-----
App v.2.4.2-d618096
Source UID: 9209
```
TypeError: Cannot read property 'x' of null
  at e.calculateBounds (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/select-tool/lib/select-tool.js:1:8095)
  at e.mousemoveDrag (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/select-tool/lib/select-tool.js:1:4971)
  at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/select-tool/lib/select-tool.js:1:240
  at t.module.exports.Emitter.emit (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/emissary/lib/emitter.js:118:15)
  at t.mousewheel (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/stage.js:1:13091)
  at HTMLCanvasElement.<anonymous> (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/stage.js:1:438)

```

**Comment by petrbrzek** [comment]
Do you know how did it happen? It'll help us a lot.

**Comment by petrbrzek** [comment]
Hello, we’ve just released Avocode 2.4.3 that should fix the issue. Let us know if the problem persists.

**Comment by Chrisriis** [comment]
Sry for late reply. It seemed to happen randomly. Thanks for hopefully fixing it ;)
